<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node name="stereo_img_server"	pkg="tf_ros_detection"      type="stereo_img_server"  	respawn="true" 	output="screen">
    </node>

    <node name="tf_detect_stereo_node"	pkg="tf_ros_detection"      type="tf_detect_stereo_node"  	respawn="true" 	output="screen">
                <remap from="cam_in" 			to="/stereo_cam_node/image_raw"/>
                <remap from="cam_left_info" 	      	to="/stereo_cam_node/left/camera_info"/>
                <remap from="cam_right_info" 		to="/stereo_cam_node/right/camera_info"/>
		<param name="root_dir"	type="string" 	value="/home/interface/softwares/tensorflow_models/"   />
		<param name="labels_dir"	type="string" 	value="/mscoco_label_map2.pbtxt" /> 
		<param name="models_dir"	type="string" 	value="/ssd_inception_v2_coco_11_06_2017/frozen_inference_graph.pb" />
 		<param name="threshold_Score" value="0.7" />
 		<param name="threshold_IOU" value="0.8" />
     </node>

</launch>
